README.md

```diff
@@ -52,8 +55,8 @@ With this bot you can create a Slack or Discord bot that is connected to your se
    
 ## Running ragtime bots
 
-RagTime supports [Slack bot](#Steps-to-create-slack-bot) or [Discord bot](#Steps-to-create-discord-bot). 
```

Comment:
maybe change "and whatsapp" to "or whatsapp"?
Also can you run 2 of them or just 1 or all 3?